contrib/freeze_packages.sh

```diff
@@ -6,33 +6,49 @@ set -e
 venv_dir=~/.electrum-venv
 contrib=$(dirname "$0")
 
+
+
+# note: we should not use a higher version of python than what the binaries bundle
+if [[ ! "$SYSTEM_PYTHON" ]] ; then
+    SYSTEM_PYTHON=$(which python3.6) || printf ""	
```

Comment:
Why is there python3.6?

Reply:
based on changes implemented in Electrum